core/src/lightclient/mod.rs

```diff
@@ -82,4 +122,61 @@ impl StarknetClient {
         let block = starknet_block_to_eth_block(starknet_block);
         Ok(block)
     }
+
+    /// Get the number of transactions in a block given a block id.
+    /// The number of transactions in a block.
+    ///
+    /// # Arguments
+    ///
+    ///
+    ///
+    /// # Returns
+    ///
+    ///  * `block_number(u64)` - The block number.
+    ///
+    /// `Ok(Bytes)` if the operation was successful.
+    /// `Err(LightClientError)` if the operation failed.
+    async fn get_code(
+        &self,
+        ethereum_address: Address,
+        starknet_block_id: StarknetBlockId,
+    ) -> Result<Bytes, LightClientError> {
+        // Convert the Ethereum address to a hex-encoded string
+        let address_hex = hex::encode(ethereum_address);
+        // Convert the hex-encoded string to a FieldElement
+        let ethereum_address_field_element = FieldElement::from_hex_be(&address_hex).unwrap();
```

Comment:
if you'd like we can rename everything that is a field element conversion with a _fe suffix
like `ethereum_address_fe`, `block_number_fe` etc. that way we have a unified way to name things

Reply:
Ok